src/less/day-slot.less

```diff
@@ -1,5 +1,10 @@
 @import './variables.less';
 
+.rbc-timeslice-group + .rbc-timeslice-group {
+  border-top: 1px solid @cell-border;
+  margin-bottom: -1px;
```

Comment:
the the negative margin isn't great, but I see that its needed until the Dayslots switch over to use these components...I would add a FIXME comment, or just include the DaySlot stuff in the PR as well.